docs/adr/adr_001_cache_bust.md

```diff
@@ -0,0 +1,322 @@
+
+`GET /buckets/main/collections/{cid}/changeset?_since={last-data-timestamp)&_expected={bumped-timestamp}`
+
+The response will contain the refreshed signature, and an empty list of changes give that the data itself wasn't changed since the last synchronization.
+
+**Complexity**
+
+Low. The overall architecture would not be modified.
+
+**Impact**
+
+High.
+
+Bumping the data timestamp on signature refresh will have a significant impact both on servers and on clients.
+
+We refresh the signature of all collections that haven't been updated in the last 7 days. With this option, we will force synchronization on all clients for these collections every at most 7 days.
+
+This will increase trafic on our servers, although the impact will be relatively low on the origins.
+
+> Note: traffic on origin shouldn't be too affected. And we are very likely to refresh signatures of several collections are once, therefore grouping them in one push notification.
+
+During synchronization, the clients will apply the empty list of changes (delta since last sync), and revalidate the signature of their local data.
+
+This will have an impact on clients CPUs on every signature refresh. But this will indirectly fix [bug 1640126](https://bugzilla.mozilla.org/show_bug.cgi?id=1640126), which consists in verifying local data when up-to-date.
+
+> Note: We could adjust the [behaviour of the lambda](https://github.com/mozilla-services/remote-settings-lambdas/blob/443117cb16cc9d720c59341c6c811e7bf127b289/commands/refresh_signature.py#L78-L93) to control the number of refreshed collections in one run.
+
+**Efforts**
+
+Low. This would be a one-liner in the `kinto-remote-settings` server plugin. Tagging and deploying is relatively cheap.
+
+**Rollout**
+
+Fast. The code change can be rolled out immediately. Effects would be visible after first signature refresh.
+
+This would cover both Desktop and Rust clients.
+
+And this option is easy to rollback.
+
+
+### Option 2 - Use metadata timestamp as cache bust
+
+This option would consist in adding a new field in the monitor/changes entries, providing the metadata timestamp.
+
+```
+GET /buckets/monitor/collections/changes/records
+
+{
+  "data": [
+    {
+      "id": "eae22ba6-631d-fcab-fa53-8a7c5adb39a9",
+      "last_modified": 1646297843179,
+      "metadata_timestamp": 1646293449944,
+      "bucket": "security-state",
+      "collection": "cert-revocations",
+      "host": "firefox.settings.services.mozilla.com"
+    },
+    ...
+  ]
+}
+```
+
+The clients would use this `metadata_timestamp` field instead of `last_modified` as the value passed in the `?_expected=` query param to bust the cache.
+
+**Complexity**
+
+Low. The overall architecture would not be modified.
+
+**Impact**
+
+High. This will **multiply by 2 the trafic on our origin servers**, since we would have a population of clients that would bust the cache with one timestamp value, and the rest with another value.
```

Comment:
```suggestion
High. This will **multiply by 2 the traffic on our origin servers**, since we would have a population of clients that would bust the cache with one timestamp value, and the rest with another value.
```